### ISSUE TITLE
Release YOLO26 on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [ultralytics/ultralytics#24689]
Hi @glenn-jocher 🤗

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your work through Hugging Face's daily papers as YOLO26 got featured: https://huggingface.co/papers/2606.03748.
The paper page lets people discuss about your paper and lets them find artifacts about it (your models for instance),
you can also claim the paper as yours which will show up on your public profile at HF, add Github and project page URLs.

Would you like to host the new YOLO26 model family you've pre-trained on https://huggingface.co/models?
I see you're currently hosting the `.pt` checkpoints on GitHub Releases. Hosting on Hugging Face (ideally under the [ultralytics](https://huggingface.co/ultralytics) organization) will give the new models more visibility and enable better discoverability through our metadata tags (allowing users to filter by task like detection, segmentation, or pose). We can also link them directly to the paper page.

If you're down, leaving a guide [here](https://huggingface.co/docs/hub/models-uploading). Since the Ultralytics ecosystem is so popular, people can also easily use [hf_hub_download](https://huggingface.co/docs/huggingface_hub/en/guides/download#download-a-single-file) to pull the weights directly into their local workflows.

After uploaded, we can also link the models to the paper page (read [here](https://huggingface.co/docs/hub/en/model-cards#linking-a-paper)) so people can discover your latest SOTA work.

You can also build a demo for YOLO26 on [Spaces](https://huggingface.co/spaces), we can provide you a ZeroGPU [grant](https://huggingface.co/docs/hub/en/spaces-gpus#community-gpu-grants), which gives you free GPU-backed compute for eligible demo Spaces.

Let me know if you're interested/need any guidance :)

Kind regards,

Niels

**Comment by UltralyticsAssistant** [comment]
👋 Hello @NielsRogge, thank you for reaching out and for your interest in Ultralytics and YOLO26 🚀! This is an automated response to let you know we’ve received your issue, and an Ultralytics engineer will also assist you soon.

We appreciate you sharing this proposal around YOLO26 model hosting and discoverability. In the meantime, we recommend a visit to the [Docs](https://docs.ultralytics.com/) where you can find many [Python](https://docs.ultralytics.com/usage/python) and [CLI](https://docs.ultralytics.com/usage/cli) usage examples and general project information.

If this issue involves a feature request or integration proposal, please provide as much context as possible, including the intended workflow, expected benefits, and any implementation details that may help the team evaluate it efficiently 🙌

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):
- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo26"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes) and [Tasks](https://docs.ultralytics.com/tasks) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by NielsRogge** [comment]
Update: it looks https://huggingface.co/Ultralytics/YOLO26 is already available, it would be great to link it to the paper page.

This can be done by mentioning https://huggingface.co/papers/2606.03748 in the README of https://huggingface.co/Ultralytics/YOLO26.

**Comment by glenn-jocher** [comment]
Thanks for the heads-up, Niels — good catch. We’ll update the `YOLO26` Hugging Face model card README to reference the paper so it links through properly.
